book/src/data-model/sboms.md

```diff
@@ -0,0 +1,18 @@
+## SBOMs
+
+The `Sbom` entity records that an SBOM was ingested or generated for a `Package` that is being
+_directly monitored_. See the [`Packages`](./packages.md) documentation for a discussion.
+
+![SBOMs](sboms.png)
+
+### Data Dictionary
+
+- `Sbom` - An SBOM is a snapshot inventory of the components that make up a piece of software at a 
```

Comment:
snapshot `manafest`...?  Just a pointer to how we decided the top level struct name for the SDL Report structure.  We went with `manafest` while I think  `inventory` was another option.

Reply:
Nice!  Updated in next push.